THIRD-PARTY-LICENSES.txt

```diff
@@ -0,0 +1,386 @@
+
+Lists of 384 third-party dependencies.
+     (BSD License) AntLR Parser Generator (antlr:antlr:2.7.7 - http://www.antlr.org/)
```

Comment:
Is there a way to get this sorted?

Reply:
The list of licenses is now sorted by artifact name